tests/e2e/utils/logger.js

```diff
@@ -32,71 +21,57 @@ const log = (...args) => {
 
         fs.writeFileSync(CONFIG.LOG_FILE, '');
     }
-    const time = new Date();
-    const timeStr = `${time.getHours()}:${time.getMinutes()}:${time.getSeconds()} ${time.getMilliseconds()}`;
-    fs.appendFileSync(CONFIG.LOG_FILE, `[${timeStr}]   ${args.join(' ')}\n`);
+    fs.appendFileSync(
+        CONFIG.LOG_FILE,
+        `${_.map(args, (arg) => {
+            if (typeof arg === 'string') {
+                // Remove color codes from arg, because they are not supported in log files
+                // eslint-disable-next-line no-control-regex
+                return arg.replace(/\x1b\[\d+m/g, '');
+            }
+            return arg;
+        })
+            .join(' ')
+            .trim()}\n`,
+    );
 };
 
-const info = (...args) => {
-    log('> ', ...args);
+const log = (...args) => {
+    const argsWithTime = [getDateString(), ...args];
+    console.debug(...argsWithTime);
+    writeToLogFile(...argsWithTime);
 };
 
-const important = (...args) => {
-    const lines = [`🟦 ${COLOR_BLUE}`, ...args, `${COLOR_RESET}\n`];
-    log(...lines);
+const info = (...args) => {
+    log('▶️', ...args);
 };
 
 const success = (...args) => {
-    const lines = [`🟦 ${COLOR_GREEN}`, ...args, `${COLOR_RESET}\n`];
+    const lines = ['✅', COLOR_GREEN, ...args, COLOR_RESET];
     log(...lines);
 };
 
 const warn = (...args) => {
-    const lines = [`\n${COLOR_YELLOW}⚠️`, ...args, `${COLOR_RESET}\n`];
+    const lines = ['⚠️', COLOR_YELLOW, ...args, COLOR_RESET];
     log(...lines);
 };
 
 const note = (...args) => {
-    const lines = [`${COLOR_DIM}`, ...args, `${COLOR_RESET}\n`];
+    const lines = [COLOR_DIM, ...args, COLOR_RESET];
     log(...lines);
 };
 
 const error = (...args) => {
-    const lines = [`\n🔴 ${COLOR_RED}`, ...args, `${COLOR_RESET}\n`];
+    const lines = ['🔴', COLOR_RED, ...args, COLOR_RESET];
     log(...lines);
 };
 
-const progressInfo = (textParam) => {
-    let text = textParam || '';
-    const getTexts = () => [`🕛 ${text}`, `🕔 ${text}`, `🕗 ${text}`, `🕙 ${text}`];
-    log(textParam);
-
-    const startTime = Date.now();
-    let i = 0;
-    const timer = setInterval(() => {
-        process.stdout.write(`\r${getTexts()[i++]}`);
-        // eslint-disable-next-line no-bitwise
-        i &= 3;
-    }, Number(LOGGER_PROGRESS_REFRESH_RATE));
-
-    const getTimeText = () => {
-        const timeInSeconds = Math.round((Date.now() - startTime) / 1000).toFixed(0);
-        return `(${COLOR_DIM}took: ${timeInSeconds}s${COLOR_RESET})`;
-    };
-    return {
-        updateText: (newText) => {
-            text = newText;
-            log(newText);
-        },
-        done: () => {
-            clearInterval(timer);
-            success(`\r✅ ${text} ${getTimeText()}\n`);
-        },
-        error: () => {
-            clearInterval(timer);
-            error(`\r❌ ${text} ${getTimeText()}\n`);
-        },
-    };
+module.exports = {
```

Comment:
@hannojg What was the reason of changing es6 `export` with `module.exports`?

Reply:
I have a TS check failing in one of my [Draft PRs](https://github.com/Expensify/App/pull/37004) after this update, so I want to make sure if it was on a purpose or just by accident